# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/nsWindow.cpp
# Commit: f37913e7cb92
# Full Hash: f37913e7cb9295617b7e5a7ac5f462f29df583b4
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-18 15:53:06
# Regressor Bug: 1726063
# File Overlap Count: 1
# Description:
#   Bug 1726063 - Remove support for non-WR compositor initialization. r=jrmuizel,aosmond
#   
#   Depends on D122796
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D122798
# ==============================================================================

diff -r 7398bc391379 -r f37913e7cb92 widget/windows/nsWindow.cpp
--- a/widget/windows/nsWindow.cpp	Tue Aug 17 23:19:44 2021 +0000
+++ b/widget/windows/nsWindow.cpp	Tue Aug 17 23:19:44 2021 +0000
@@ -4109,7 +4109,7 @@
         reinterpret_cast<uintptr_t>(static_cast<nsIWidget*>(this)),
         mTransparencyMode, mSizeMode);
     // If we're not using the compositor, the options don't actually matter.
-    CompositorOptions options(false, false, false);
+    CompositorOptions options(false, false);
     mBasicLayersSurface =
         new InProcessWinCompositorWidget(initData, options, this);
     mCompositorWidgetDelegate = mBasicLayersSurface;
